packages/rum-recorder/src/transport/send.ts

```diff
@@ -0,0 +1,31 @@
+import { HttpRequest, objectEntries } from '@datadog/browser-core'
+import { SegmentMeta } from '../types'
+
+export const SEND_BEACON_BYTE_LENGTH_LIMIT = 60_000
```

Comment:
While being used in a same way, those values are very different, and using a value a lot lower would increase the number of segment consequently. I agree, it would be nice to use the same limit though. Could you remind me why the `core` constant is only 16KB?

Reply:
IIRC the limit is on the sendBacon queue that is shared across sendBeacon usages on the page.
The intent was to limit our usage to allow other sendBeacon users to send data at the same time (especially for page close).
FMU, setting this limit too high here could even bother rum requests.

Reply:
FTR: keep 60k for now, explore potential issues in a separate task